npm/private/utils.bzl

```diff
@@ -117,6 +117,26 @@ def _is_at_least_bazel_6():
     # native.bazel_version only works in repository rules.
     return "apple_binary" not in dir(native)
 
+def _parse_package_name(package):
+    # Parse a @scope/name string and return a [scope, name] list
```

Comment:
nit: "return a (scope, name) tuple"

Reply:
My bad, I've had it like you suggested but I found a similar function that was using the same description: https://github.com/aspect-build/rules_js/blob/724c377bfed9db82881b4174b9652700f9c426c7/npm/private/utils.bzl#L33-L39

Should I change its description too?

Reply:
Yes please. Nice catch. Looks like that one got out of sync.